editor/debugger/editor_debugger_inspector.cpp

```diff
@@ -280,7 +280,28 @@ String EditorDebuggerInspector::get_stack_variable(const String &p_var) {
 	for (KeyValue<StringName, Variant> &E : variables->prop_values) {
 		String v = E.key.operator String();
 		if (v.get_slice("/", 1) == p_var) {
-			return variables->get_variant(v);
+			Variant var = variables->get_variant(v);
+
+			String prefix;
+			String literal = String(var);
+
+			switch (var.get_type()) {
+				case Variant::STRING:
+					literal = literal.quote("\"");
+					break;
+				case Variant::STRING_NAME:
+					prefix = "&";
+					literal = literal.quote("\"");
```

Comment:
```suggestion
					literal = "&" + literal.quote("\"");
```
Same in NODE_PATH.
The prefix variable could be removed, since it is only used in two cases.